PWA/package.json

```diff
@@ -11,9 +11,8 @@
     "start": "node server.js",
     "test": "jest --setupFiles dotenv/config"
   },
-  "dependencies": {  
+  "dependencies": {
     "@babel/core": "^7.10.1",
-    "@vue/cli-service": "^4.4.1",
```

Comment:
NB: this line still exists, npm just re-ordered the dependencies alphabetically.

Reply:
weird